subsys/net/lib/nrf_cloud/src/nrf_cloud_fsm.c

```diff
@@ -701,6 +701,7 @@ static int dc_rx_data_handler(const struct nct_evt *nct_evt)
 
 	switch (nrf_cloud_dc_rx_topic_decode(cloud_evt.topic.ptr)) {
 	case NRF_CLOUD_RCV_TOPIC_AGPS:
```

Comment:
Is it necessary to keep the AGPS case, defines etc.? I would think that nothing is received on the AGPS topic if the device does not send a request for it.

Reply:
Good catch, thanks.